CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+## [0.0.1] - 2020-07-01
```

Comment:
```suggestion
## [0.0.1]

* July 1st, 2020*
```

In keeping with tendermint-rs [changes.md](https://github.com/informalsystems/tendermint-rs/blob/master/CHANGES.md), we should consider writing the date between parentheses. The "Github Repo Standards", however, has a different proposal: to write the release date in italics on a separate line (see merge_requests/47 of the handbook). I guess there is no right or wrong way, and this is not a critical aspect, but it would be helpful to have a single, consistent format across our projects. I would vote for the repo standards format.